broker-cli/commands/wallet.js

```diff
@@ -529,6 +530,35 @@ async function unlock (args, opts, logger) {
   }
 }
 
+/**
+ * Recover a wallet from a provided mnemonic seed and password. backup file is optional
+ *
+ * ex: `sparkswap wallet recover <symbol>`
+ *
+ * @function
+ * @param {Object} args
+ * @param {string} args.symbol
+ * @param {Object} opts
+ * @param {Logger} logger
+ * @returns {void}
+ */
+async function recover (args, opts, logger) {
+  const { symbol } = args
+  const { rpcAddress = null } = opts
+
+  try {
+    const client = new BrokerDaemonClient(rpcAddress)
+
+    const password = await askQuestion(`Please enter your wallet password:`, { silent: true })
+    const seed = await askQuestion(`Please enter your 24 word recover seed, with each word separated by spaces`, { silent: true })
+    const backupPath = await askQuestion(`Enter the path to your backup file (optional)`)
```

Comment:
Why is this optional?

Reply:
from the CLI, we should force the user to provide this. I haven't made the change yet